### ISSUE TITLE
Navbar title color differs on Android and Windows

### ISSUE THREAD

**Issue by david-maw** [dotnet/maui#6761]
### Description

It appeared that the page title in the NavBar was missing on Android but it actually wasn't, it just defaults to white text on a white background whereas Windows defaults to black text on a white background.

### Steps to Reproduce

1. Unzip the project from
[MultiPage-titlecolor.zip](https://github.com/dotnet/maui/files/8605727/MultiPage-titlecolor.zip)

2. Compile and run it on Windows, you should see this:
![multipage-titlecolor-windows-mainpage](https://user-images.githubusercontent.com/12423046/166316719-4fa9bcef-35a1-4700-9bb4-f0212eb043a3.jpg)

3. Compile and run it on Android, you should see this:
![multipage-titlecolor-android-mainpage](https://user-images.githubusercontent.com/12423046/166317394-99e73483-bf50-48fc-8f8e-8e75aa967789.jpg)

Note the apparent lack of a hamburger icon or page title on Android - they are there, just drawn in white on a white background. Explicitly setting `Shell.TitleColor="Black"` will make the title show up, but the hamburger icon is still white on a white background. Alternatively you can set `Shell.BackgroundColor` to reveal them on Android but it seemingly has no effect on Windows.

### Version with bug

Release Candidate 2 (current)

### Last version that worked well

Unknown/Other

### Affected platforms

Android

### Affected platform versions

Android 11

### Did you find any workaround?

No

**Comment by YingFengTingYu** [comment]
In light mode, I use color "CornflowerBlue" and it works well, but when I use color "Black" it doesn't work(and the real color is "White"). The color "Black" works well when I use dark mode.

**Comment by jfversluis** [comment]
I indeed suspect that this has something to do with Dark Theme. @david-maw could you confirm you have this running in Dark Theme on Android? What happens if you switch to light theme?

Nevertheless this needs fixing, just trying to get a bit of extra information here

**Comment by david-maw** [comment]
@jfversluis No, I just test on a clean Android emulator which defaults to "dark theme" off.

**Comment by XamlTest** [comment]
Verified this on Visual Studio Enterprise 17.6.0 Preview 5.0. This issue can be reproduced with below project:

Navbar title color differs on Android and Windows。
Windows 11:
![image](https://user-images.githubusercontent.com/97647558/234515905-dfde23ca-19bc-437d-a7e8-568b50644d49.png)

Android:
![image](https://user-images.githubusercontent.com/97647558/234516673-5ddc9018-6168-4e21-916a-f1c47a77dd18.png)

**Comment by Shalini-Ashokan** [comment]
@david-maw, If the background color is explicitly set to white, the title text becomes invisible.
To keep the title visible, set Shell.TitleColor="Black". Similarly, to change the color of the hamburger icon, set Shell.ForegroundColor="Black"

**Comment by david-maw** [comment]
Thanks @Shalini-Ashokan, that seems like a simple workaround for anyone who's stuck on this.
